ehr/resources/web/ehr/overrides.js

```diff
@@ -0,0 +1,32 @@
+
+
+LDK.Utils.splitIds = function(subjectArray, unsorted, preserveDuplicates) {
+    if (!subjectArray){
+        return [];
+    }
+
+    subjectArray = Ext4.String.trim(subjectArray);
+    subjectArray = subjectArray.replace(/[\s,;]+/g, ';');
+    subjectArray = subjectArray.replace(/(^;|;$)/g, '');
+
+    if (subjectArray){
+        subjectArray = subjectArray.split(';');
+    }
+    else {
+        subjectArray = [];
+    }
+
+    subjectArray = EHR.Utils.formatAnimalIds(subjectArray);
+
+    if (subjectArray.length > 0) {
```

Comment:
this is picky, but shouldnt this test greater than 1 rather than zero? do we actually need this check at all?

Reply:
Yeah true I updated to greater than 1. The check just saves a little processing.